_pytest/test_sanity.py

```diff
@@ -63,11 +63,16 @@ def import_tracking(name, *x, **xs):
         builtins.__import__ = import_tracking
 
     # import all available modules and track imports on the way
-    import_submodules("rasa_nlu", skip_list={"rasa_nlu.wsgi"})      # wsgi needs to be skipped as it instantiates an app
+    import_submodules("rasa_nlu", skip_list={})
 
     def find_modules_importing(name):
         return {v for k, vs in tracked_imports.items() if k.startswith(name) for v in vs}
 
     assert not find_modules_importing(banned_package), \
         "No module should import {} globally. Found in {}".format(
             banned_package, ", ".join(find_modules_importing(banned_package)))
+
+    if PY2:
+        __builtin__.__import__ = original_import_function
+    else:
+        builtins.__import__ = original_import_function
```

Comment:
Not necessary at all but as Py.test behaves sometimes strangely, better make things clear.